collectors/perf.plugin/perf_plugin.c

```diff
@@ -527,11 +529,43 @@ static void perf_send_metrics() {
 
     // ------------------------------------------------------------------------
 
+    if(likely(perf_events[EV_ID_INSTRUCTIONS].updated) && likely(perf_events[EV_ID_CPU_CYCLES].updated)) {
+        if(unlikely(!ipc_chart_generated)) {
+            ipc_chart_generated = 1;
+
+            printf("CHART %s.%s '' '%s' 'instructions/cycle' %s '' line %d %d '' %s\n"
+                   , RRD_TYPE_PERF
+                   , "instructions_per_cycle"
+                   , "Instructions per Cycle(IPC). An IPC < 1.0 likely means memory bound, and an IPC > 1.0 likely means instruction bound. For more details about the metric take a lookt at this <a href=\"https://www.brendangregg.com/blog/2017-05-09/cpu-utilization-is-wrong.html\" target=\"_blank\">blog post</a>."
```

Comment:
Usually, we use dashboard_info.js for that. Let's move everything after `Instructions per Cycle(IPC)` to it.

Reply:
I push another commit moving content for dashboard_info.js.